source/sync/migrate-sync-modes.txt

```diff
@@ -26,6 +26,10 @@ Partition-Based Sync.
 Requirements
 ------------
 
+Currently, the Kotlin and Swift SDKs can migrate from Partition-Based Sync to
+Flexible Sync. The C++, Flutter, Java, .NET, Node.js, and React Native SDKs
```

Comment:
can remove flutter from this list since migration is irrelevant there. i'm also not sure if the c++ sdk supports PBS? can't find anything in the docs indicating that it does

Reply:
Correct - Flutter and C++ are the newest SDKs and only support flexible sync.